### ISSUE TITLE
Impossible to override FacebookAuthButton TextStyle

### ISSUE THREAD

**Issue by Thithip** [elbeicktalat/flutter_auth_buttons#17]
Hi!

I'm trying to override the TextStyle property for the FacebookAuthButton to reduce the FontSize. It works well with other button, like Google one, but not the Facebook button.

I think this is due to the overritten method `getTextStyle()` in the FacebookAuthButton widget, which ignore the `textStyle` attribute given:
```dart
  @override
  TextStyle getTextStyle() {
    if (iconStyle == AuthIconStyle.secondary && style == null)
      return TextStyle(
        color: darkMode ? Colors.white : Colors.blue[800],
        fontSize: 18,
        fontWeight: FontWeight.bold,
        letterSpacing: 0.50,
      );
    return TextStyle(
      color: Colors.white,
      fontSize: 18,
      fontWeight: FontWeight.bold,
      letterSpacing: 0.50,
    );
  }
```

ref. : https://github.com/elbeicktalat/flutter_auth_buttons/blob/master/lib/res/buttons/facebook_auth_button.dart#L97-L112

**Comment by elbeicktalat** [comment]
@Thithip Thanks for reporting this issue,
I have fixed it and I'll push a new version of the package soon,  for sure it will be included a new features.

**Comment by Thithip** [comment]
Thanks for the very quick fix 🚀✌️

**Comment by sabetAI** [comment]
I'm experiencing the same issue with the `EmailAuthButton` using `auth_buttons: ^1.0.1+3`.

**Comment by elbeicktalat** [comment]
> I'm experiencing the same issue with the `EmailAuthButton` using `auth_buttons: ^1.0.1+3`.

Fixed with version 1.0.1+4

**Comment by elbeicktalat** [comment]
> I'm experiencing the same issue with the `EmailAuthButton` using `auth_buttons: ^1.0.1+3`.

Fixed with version 1.0.1+4
